crates/red_knot_test/src/lib.rs

```diff
@@ -95,6 +109,23 @@ pub fn run(
     assert!(!any_failures, "Some tests failed.");
 }
 
+/// Defines the format in which mdtest should print an error to the terminal
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum OutputFormat {
+    /// The format `cargo test` should use by default.
+    Cli,
+    /// A format that will provide annotations from GitHub Actions
+    /// if mdtest fails on a PR.
+    /// See <https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-error-message>
+    GitHub,
+}
+
+impl OutputFormat {
+    const fn is_cargo(self) -> bool {
```

Comment:
```suggestion
    const fn is_cli(self) -> bool {
```

Reply:
hmm, maybe I should just derive `is_macro::Is` ;)